stable/mtls-proxy/values.yaml

```diff
@@ -6,7 +6,7 @@ replicaCount: 1
 
 image:
   repository: squareup/ghostunnel
-  tag: v1.5.3
+  tag: v1.5.2
```

Comment:
It looks like ghostunnel moved to a new repo, https://github.com/square/ghostunnel now redirects to ghostunnel/ghostunnel. So they moved the tag to a new corresponding dockerhub account: https://hub.docker.com/layers/ghostunnel/ghostunnel/v1.5.3/images/sha256-0f37069c6825b54211bd13f4f594318a454f31e65be90f46724c402451c37532?context=explore. I think we can change the repository to `ghostunnel/ghostunnel` and leave the tag in place.

Reply:
oh nice find! @martinhanzik @hectorj2f I'll make the change to use the new repo and keep the image at the newer v1.5.3